packages/algob/src/types.ts

```diff
@@ -1,9 +1,4 @@
-import type {
-  Account, AccountMap,
-  ASADeploymentFlags,
-  SSCDeploymentFlags, SSCOptionalFlags,
-  TxParams
-} from "@algorand-builder/runtime/build/types";
+import * as runtime from "@algorand-builder/runtime";
```

Comment:
we can add a variable to make things shorter:
```
const rtypes = runtime.types
```

Reply:
ok